src/eru.Infrastructure/Persistence/ApplicationDbContext.cs

```diff
@@ -12,16 +12,26 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : ba
             
         }
         
-        [ExcludeFromCodeCoverage]
         public DbSet<Class> Classes { get; set; }
 
+        public DbSet<User> Users { get; set; } 
+
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            modelBuilder.Entity<Class>()
-                .HasKey(x=>x.Name);
-            modelBuilder.Entity<Class>()
-                .Property(x => x.Name)
-                .HasMaxLength(255);
+            modelBuilder.Entity<Class>(x =>
+            {
+                x.HasKey(x => x.Name);
```

Comment:
'X' has already other meaning (x here represents entitymodelbuilder not class entity).
You should replace inner x with y's for example.